Makefile

```diff
@@ -13,7 +15,10 @@ man:
 	ronn --roff --pipe --manual RMT MANUAL.md > rmt-cli.8 && gzip -f rmt-cli.8
 	mv rmt-cli.8.gz package/obs
 
-dist: clean man
+dist:
+	docker-compose run -it -v $(PWD):/srv/www/rmt --rm rmt make build-tarball
```

Comment:
What's the reason why we need to build the  dist tarball with docker-compose?

Reply:
The reason was that multiple people have problems with their ruby2.5 setup and this decouples the packaging process from the development setup. So you get a tarball no matter what your development setup looks like. Do you see a problem?

Reply:
I see. Well for me it's inconvenient because I need to build containers now to build the tarball (took 5m). But I see that I can also use `make build-tarball`.